### PR TITLE
Readability and documentation

### PR DESCRIPTION
**Pull request by lemonzi**
I split #14 in chunks because the conflicts were getting messy.

What do you think about `quantization_steps`? Should it have `_channels` as a suffix, like `encoding_channels` or `quantization_channels`?

### CODE REVIEW

**Comment by ibab** [comment]
Can you also adjust the tests to work with the modified code?
And I like `quantization_channels` the most, but I don't have a strong preference.

File: wavenet_ops.py [hunk 1]
```diff
@@ -20,8 +19,8 @@ def batch_to_time(value, dilation, name=None):
         return tf.reshape(transposed, [tf.div(shape[0], dilation), -1, shape[2]])
 
 
-def causal_conv(value, filter_, dilation, name=None):
-    with tf.name_scope('causal_conv'):
+def causal_conv(value, filter_, dilation, name='casual_conv'):
```
Comment (ibab) [inline wavenet_ops.py]: `casual_conv` -> `causal_conv` 😀

Comment (lemonzi) [inline wavenet_ops.py]: Oops!

Comment (lemonzi) [inline wavenet_ops.py]: casual

**Pull request merged**